test/TestAsset/NullDefaultHintsClass.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
```

Comment:
Docblocks are not important here, and may actually confuse the reader of the assets, so maybe I should remove it from `EmptyClass` as well.